src/app/character.ts

```diff
@@ -295,9 +296,9 @@ export class Character implements Loadable {
   }
 
   displayHeight(){
-    if(!this.height)
+    if(!this.height_foot && !this.height_inch)
```

Comment:
What if a character is 5'0 or 6'0?

Reply:
Got it! made change to support 0 in either feet or inch section.

Reply:
take a peek at the second commit!